__tests__/src/util/iterationDecorator-test.js

```diff
@@ -1,61 +1,45 @@
-/* eslint-env mocha */
-import expect from 'expect';
-import iterationDecorator from '../../../src/util/iterationDecorator';
+import test from 'tape';
```

Comment:
I'm new to tape. Is the primary goal to support something for old Node, or are there other reasons for migrating? I understand that maintaining two test runners would be a huge headache. But I want to understand this change more before giving it a rubber stamp.

Reply:
The primary goal is to ensure all supported nodes are tested, and tape afaik is the only testing tool for which the latest version allows this.

Separately, i do actually prefer this testing approach, since it’s just JavaScript and avoids a lot of the complexity, performance issues, and edge cases of jest.

Reply:
Alright. I suppose the test runner could always be changed again later when appropriate. How does Vitest fare for these requirements?

Reply:
vitest requires Node 18. I've had really good experiences with it

I've also used uvu a fair amount, which only requires Node 8.

I think there's probably another solution here simpler than switching out the whole test runner though. If you want to check that you're not using APIs only available in newer versions of Node, that's something that could probably be accomplished with type checking or linting.

Reply:
Yes, that’s typical, it’s just TAP output. It can be piped to any TAP reporter to make it nicer.

It’s about for more than just statically analyzable stuff; there’s no substitute for actually running tests in supported versions.